CONTRIBUTING.md

```diff
@@ -188,7 +188,53 @@ with one another are rolled up.
 Speaking of tests, Rust has a comprehensive test suite. More information about
 it can be found [here][rctd].
 
-### External Dependencies
+### External Dependencies (subtree)
+
+As a developer to this repository, you don't have to treat the following external projects
+differently from other crates that are directly in this repo:
+
+* none so far, see https://github.com/rust-lang/rust/issues/70651 for more info
+
+They are just regular files and directories. This is in contrast to `submodule` dependencies
+(see below for those).
+
+If you want to synchronize or otherwise work with subtrees, install the `git subtree` command via
+instructions found at https://github.com/ingydotnet/git-subtree
```

Comment:
Hmm, `git subtree` is already installed on my local copy of git (Ubuntu 18.04). Is it needed to install it from GitHub?

Reply:
Also https://github.com/ingydotnet/git-subtree is was last updated 2011. It says "experimental" in the description. And it is a fork of https://github.com/matthoffman/git-subtree which is a fork of https://github.com/bibendi/git-subtree which is a fork of https://github.com/apenwarr/git-subtree. Is that really the one we want?^^

Reply:
This is just a consequence of `s/subrepo/subtree/`, nothing to see here.
(The whole paragraph needs to be removed.)

Reply:
Ah, `subtree` looks to be upstream these days: https://github.com/git/git/blob/master/contrib/subtree/git-subtree.txt